template/_prettierrc.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
+  arrowParens: 'avoid',
```

Comment:
Here also, please undo the re-sort, so it's consistent with the default RN template.

Reply:
Okay, I'll undo this, and we'll wait until this arrives in the next release of `react-native`.